features/other_b/skin.feature

```diff
@@ -106,7 +35,8 @@ Feature: creating and editing skins
     And I should see "#title {" within "style"
     And I should see "text-decoration: blink;" within "style"
 
-  Scenario: A user should be able to select one of their own non-public skins to use in their My Skins page
+  Scenario: A user should be able to select one of their own non-public skins to use in
+  their My Skins page
   Given I am logged in as "skinner"
```

Comment:
If it does work it makes it harder to see where the tests start. Maybe a blank line ?